src/hooks/useValidation/validators.js

```diff
@@ -482,6 +482,10 @@ export const validateLccnDuplication = async ({
       query: `(${lccnQuery})${idQuery}`,
     };
 
+    if (marcType === MARC_TYPES.BIB) {
+      searchParams.query += ' and (staffSuppress=="false" or discoverySuppress=="false")';
+    }
+

```

Comment:
Are you confident these fields are always defined and always have a boolean value? If they are, great, this looks correct. But if these fields are optional, then I _think_ the correct formulation is 
```
and (not (staffSuppress=="true" and discoverySuppress=="true"))
```
but please check.

Reply:
The request `/search/instances` returns these fields with the value `false`, even if we created the record without them. But I changed the query because your suggestion is safer.